backend/src/services/secret/secret-service.ts

```diff
@@ -2081,7 +2086,7 @@ export const secretServiceFactory = ({
 
     return {
       ...updatedSecret[0],
-      tags: [...existingSecretTags, ...tags].map((t) => ({ id: t.id, slug: t.slug, name: t.name, color: t.color }))
+      tags: [...existingSecretTags, ...tags].map((t) => ({ id: t.id, slug: t.slug, color: t.color }))
```

Comment:
Did we miss the name tag here?

Reply:
Still not sure where this endpoint is used? May daniel has better idea over this one. But i think yah i missed it. 

Not used in terraform